src/contractMonitor.ml

```diff
@@ -0,0 +1,273 @@
+(* This file is part of the Kind 2 model checker.
```

Comment:
Since the contract monitor shares most of its code with the interpreter, I'd rather introduce a flag in the interpreter to run it as the contract monitor than create a separate module for it.